sdcm/results_analyze.py

```diff
@@ -416,13 +416,17 @@ class PerformanceResultsAnalyzer(BaseResultsAnalyzer):
 
     PARAMS = TestStatsMixin.STRESS_STATS
 
-    def __init__(self, es_index, es_doc_type, send_email, email_recipients, logger=None):  # pylint: disable=too-many-arguments
+    def __init__(self, es_index, es_doc_type, send_email, email_recipients, logger=None, performance_email_template=None):  # pylint: disable=too-many-arguments
```

Comment:
how this change is relevant ? to sub_tests ?

Reply:
since the template is hardcoded in `check_regression_with_subtest_baseline()`, not much point of change this...

Reply:
removed